src/base/MemoizableArray.php

```diff
@@ -57,13 +52,12 @@ class MemoizableArray implements IteratorAggregate, Countable
     /**
      * Constructor
      *
-     * @param array $elements The items to be memoized
+     * @param array $_elements The items to be memoized
```

Comment:
This is a breaking change. Someone might be specifying the argument by name.

```php
new MemoizableArray(elements: [
    // ...
]);
```

Reply:
@brandonkelly sure, I updated to skip when property vs param name are different on next commits

https://github.com/craftcms/cms/pull/16711/commits/de6fed397636a70821b072a994c20b7c8a25b6a6

and

https://github.com/craftcms/cms/pull/16711/commits/cadbf1b315548bd9f97b34c8a4acf91282d00a95

 👍